packages/react-native/src/builders/run-android/schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "title": "Run iOS application",
```

Comment:
Hi, it seems there is a typo. This schema is for Android but here is "iOS"

Reply:
Good catch, I'll fix it.